qemu/tests/rebase_negative_test_on_iscsi.py

```diff
@@ -0,0 +1,120 @@
+import re
+import logging
+
+from virttest import qemu_storage
+from virttest import lvm
+from virttest import error_context
+from avocado.utils import process
+from autotest.client.shared import error
+
+
+class NegativeRebase(qemu_storage.QemuImg):
+
+    def __init__(self, test, params, env):
+        self.test = test
+        self.params = params
+        self.env = env
+        self.data_dir = "/dev/%s" % params.get("vg_name")
+        self.lvmdevice = None
+
+    def setup_lvm(self):
+        """
+        Setup a lvm environment and create logical volume:
+
+        :return logical volume object
+        """
+        self.params["pv_name"] = self.params.get("image_name")
+        lvmdevice = lvm.LVM(self.params)
+        error_context.context("Setup lvms", logging.info)
+        for lv_name in self.params.objects("lv_name_list"):
+            self.params["lv_name"] = lv_name
+            lvmdevice.setup()
+        self.lvmdevice = lvmdevice
+
+    def start_test(self):
+        """
+        Main function to run the negative test of rebase:
+        """
+        msg = ""
+        trigger_negative = False
+        rebase_chain = self.params.get("rebase_list", "").split(";")
+        self.setup_lvm()
+        error_context.context("Create images on lvms", logging.info)
+        for image_name in self.params.objects("images"):
+            filename = "%s/%s" % (self.data_dir,
+                                  self.params["image_name_%s" % image_name])
+            self.params.update(
+                {"image_name_%s" % image_name: filename,
+                 "image_size_%s" % image_name: self.params["lv_size"]})
+            t_params = self.params.object_params(image_name)
+            qemu_image = qemu_storage.QemuImg(t_params,
+                                              self.data_dir, image_name)
+            logging.info("Create image('%s') on %s." %
+                         (image_name, qemu_image.storage_type))
+            qemu_image.create(self.params)
+        error_context.context("Rebase snapshot to backingfile", logging.info)
+        for images in rebase_chain:
+            images = map(lambda x: x.strip(), images.split(">"))
```

Comment:
The value of images is changed. But it will not impact the loop. I will change if you insist on the name is inappropriate.
For example, rebase_list = "sn2 > image1; sn1 > image1":
In the first loop, the initial value of images is a string "sn2 > image1" and will be changed as list ['sn2', 'image1'] 
In the second loop, the initial value of images is a string "sn1 > image1" and will be changed as list ['sn1', 'image1']